docs/FAQs.md

```diff
@@ -218,3 +218,11 @@ If you know of any accessibility organization that's published more formal recom
 You can adjust the tab size that GitHub uses to display files from your [account settings page](https://github.com/settings/appearance#tab-size-heading) (the default is 8 spaces).
 
 If you really want spaces in your project you can always remove the `"useTabs": true`.
+
+## How can I use `bin`?
+
+The `--bin` option allows you to create a `package.json` bin value to include for npx-style running.
+An example of this would be `"./bin/index.js"`.
+You'll need to create the folders and files that `bin` references.
+
+If you'd like an example of what that looks like, just take a look at the [CTA source code](https://github.com/JoshuaKGoldberg/create-typescript-app/blob/e7fafcb8968f8f6c551ab0917c9a6a849a3cba28/bin/index.js)!
```

Comment:
```suggestion
If you'd like an example of what that looks like, take a look at the [CTA source code](https://github.com/JoshuaKGoldberg/create-typescript-app/blob/e7fafcb8968f8f6c551ab0917c9a6a849a3cba28/bin/index.js)!
```

https://tipsforteachers.substack.com/p/59-stop-saying-the-word-just?open=false#%C2%A7a-tip-to-try-in-class-this-week